### DISCUSSION TITLE
C++ and VScode integration

### DISCUSSION THREAD

**Discussion by TamirRozenfeld** [microsoft/vscode-cpptools#9778] [Q&A] [answered]
Tried my whole day to integrate between vscode and c++ with no success i get output [Done] exited with code=1 in 0.124 seconds in VScode when i try to write c++ code,

My input:

#include <iostream>

int main() {
    std::cout << "Hello World!";
    return 0;
{
I have downloaded VScode and worked through this guide https://code.visualstudio.com/docs/cpp/config-mingw

I have created a path as explained in the guide 
<img width="374" alt="Screenshot 2022-08-23 222708" src="https://user-images.githubusercontent.com/108065131/186254174-f50ec440-3a84-4f57-a00e-337ea42c2d3a.png">
<img width="410" alt="Screenshot 2022-08-23 222732" src="https://user-images.githubusercontent.com/108065131/186254193-0c3164ae-b859-409e-b961-4586839f8ac7.png">

In the command prompt i typed

pacman -S --needed base-devel mingw-w64-x86_64-toolchain

in the command prompt when i type g++ --version gdb --version

i get this

""g++ (Rev1, Built by MSYS2 project) 12.2.0 Copyright (C) 2022 Free Software Foundation, Inc. This is free software; see the source for copying conditions. There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE."" In additional i have download the extension in VScode (runner and c++/c

Would like to get help here so i can start using c++ in VScode I am using windows 10

**Comment by sean-mcmanus** [comment, marked as answer]
Maybe uninstall/disable the Runner extension? What step in the walkthrough isn't working?

**Reply by TamirRozenfeld** [reply]
Ty very much it works!
I  uninstalled runner than restart vscode than installed and restarted vscode again.
